accessibility.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="initial-scale=1,maximum-scale=1,user-scalable=no"
+    />
+    <title>Plan Your Active Travel Scheme</title>
```

Comment:
Should the title of each page differ? It may not matter much in practice -- even if it's "Plan Your Active Travel Scheme -- Browse Schemes", in most browsers with a few tabs open, something that long won't be very visible anyway

Reply:
I don't mind, happy to make this change but like you say, I imagine it won't be visible a lot of the time